### PR TITLE
Implemening a Map component

### LINKED ISSUE #8: Add "Map"-Page
- [x] Mapcontainer

### LINKED ISSUE #31: Research Map extensions

### LINKED ISSUE #52: Display correct Map on correct page

### PR DESCRIPTION
**Pull request by lauravikanis**
Trying to display a Map on a certain Page with react-leaftlet
close #31  close #8  close #52

### CODE REVIEW

File: client/src/pages/Location.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 import PageHeadline from "../components/Header/PageHeadline";
-import placeholder from "../assets/placeholder.jpeg";
-import ImageCard from "../components/Card/ImageCard";
-import mappath from "../assets/map.svg";
+// import placeholder from "../assets/placeholder.jpeg";
+// import ImageCard from "../components/Card/ImageCard";
 
-import { Link, useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import { useQuery } from "react-query";
```
Comment (mhstrkmp) [inline client/src/pages/Location.js:8]: Maybe you could use also `QueryCache, ReactQueryCacheProvider`

See Example:
- https://react-query.tanstack.com/docs/examples/simple

---

There is also a great dev tool for the react-query library:
- https://react-query.tanstack.com/docs/devtools

**Review by mhstrkmp — APPROVED** [review]
lgtm 🚀 

---

On your home page there is the back button in your headline. Not shure if it belongs there ...

**Comment by lauravikanis** [comment]
> There seems to bee an issue with your api call. Your app want's to ` GET /api//tour/electronic`
> Behind the api ther are two `//`Maybe that's the issue.
> 
> Bildschirmfoto 2020-12-11 um 09 52 09

Good times! Thanks for the feedback. Deployment seems to be working after I changed API routing 🙌

**Comment by mhstrkmp** [comment]
> > There seems to bee an issue with your api call. Your app want's to ` GET /api//tour/electronic`
> > Behind the api ther are two `//`Maybe that's the issue.
> > Bildschirmfoto 2020-12-11 um 09 52 09
> 
> Good times! Thanks for the feedback. Deployment seems to be working after I changed API routing 🙌

Yes it works now. Looks awesome with the maps!
Congrats 👏

**Pull request merged**